proofOfConcept/ColorPickerAlpha/ColorPickerAlpha/MainWindow.xaml.cs

```diff
@@ -24,5 +24,10 @@ public MainWindow()
         {
             InitializeComponent();
         }
+
+        private void Button_Click(object sender, RoutedEventArgs e)
```

Comment:
Maybe keep the C# naming convention, ButtonClick

Reply:
Not sure, it was automatically generated for me. 0-0